packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -58,6 +58,33 @@ function toTimestamp(date) {
   return ms === 0 ? null : Math.round(ms / 1000)
 }
 
+// https://github.com/xenserver/xenadmin/blob/093ab0bcd6c4b3dd69da7b1e63ef34bb807c1ddb/XenModel/XenAPI-Extensions/VM.cs#L773-L827
+const getVmGuestToolsProps = vm => {
+  const { $metrics: metrics, $guest_metrics: guestMetrics } = vm
+  if (!isVmRunning(vm) || metrics === undefined || guestMetrics === undefined) {
+    return {}
+  }
+
+  const { major, minor } = guestMetrics.PV_drivers_version
+  const hasPvVersion = major !== undefined && minor !== undefined
+  const pvDriversVersion = hasPvVersion ? +`${major}.${minor}` : undefined
+
+  // "PV_drivers_detected" field don't exists on XS < 7
+  const pvDriversDetected = guestMetrics.PV_drivers_detected ?? hasPvVersion
+
+  return {
+    // Linux VMs don't have the flag "feature-static-ip-setting"
```

Comment:
The first point can be verified in [`xapi project web site`](https://xapi-project.github.io/xen-api/classes/vm_guest_metrics.html).

There is no official documentation for the second point but `xenAdmin` doesn't base on `feature-static-ip-setting` in case of linux VMs.

Reply:
`PV_drivers_detected` property is only used on line https://github.com/vatesfr/xen-orchestra/pull/5007/files#diff-cbe526e73142a277b38a628ece1c6b57R72 why should i regroup it with `managementAgentDetected`?